jerry-core/parser/regexp/re-parser.h

```diff
@@ -78,7 +78,7 @@ typedef enum
 /**
  * Undefined character (out of the range of the codeunit)
  */
-#define RE_CHAR_UNDEF 0xFFFFFFFF
+#define RE_CHAR_UNDEF 0xFFFF
```

Comment:
We should certainly not use `LIT_UTF16_CODE_UNIT_MAX` in place of `RE_CHAR_UNDEF`, since -- as its name suggests -- it is a type domain limit. We might have a `#define LIT_CHAR_NONCHAR ((ecma_char_t) 0xFFFF)` -- perhaps in lit-char-helpers.h instead of the previously suggested lit-strings.h (now seems to fit there better).